vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/resources/META-INF/resources/frontend/vaadin-big-decimal-field.js

```diff
@@ -13,57 +13,58 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
+import { TextField } from '@vaadin/text-field';
```

Comment:
I think this be `@vaadin/text-field/src/vaadin-text-field.js` to allow using Material theme.

Reply:
Thanks, fixed.